docker-compose.yml

```diff
@@ -0,0 +1,37 @@
+etcd:
+    image: autopilotpattern/etcd
+    mem_limit: 128m
+    ports:
+      - 4001
+      - 2379
+      - 2380
+    links:
+      - bootstrap:bootstrap
+    labels:
+      - triton.cns.services=etcd
+    environment:
+      - DISCOVERY
+      - GOMAXPROCS=2
+
+bootstrap:
+    image: autopilotpattern/etcd
+    mem_limit: 128m
+    ports:
+      - 4001:4001
+      - 2379:2379
+      - 2380:2380
+    hostname: bootstrap
+    labels:
+      - triton.cns.services=etcd
```

Comment:
I'm still reading through, but marking this to come back to: should this advertise the `etcd` service, or...?